release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -240,24 +241,24 @@ ddpg-hopperbulletenv-v0:
         min_sample_timesteps_per_reporting: 1000
         target_network_update_freq: 0
         tau: 0.001
-        buffer_size: 10000
-        prioritized_replay: True
-        prioritized_replay_alpha: 0.6
-        prioritized_replay_beta: 0.4
-        prioritized_replay_eps: 0.000001
+        replay_buffer_config:
+          capacity: 10000
```

Comment:
Should we add type here either way? Just to make it more clear which class is used?

Reply:
Ok, I've added explicit types to all yamls.